### DISCUSSION TITLE
`which-function-mode`  not honored with `consult-line` together with `consult-preview-at-point-mode` (in `completion-list-mode`)

### DISCUSSION THREAD

**Discussion by emacksnotes** [minad/consult#994] [Q&A]
The sscreenshot below is self-explanatory.

I want a recipe which updates `which-function` in `mode-line` ...

**_`which-function-mode`  not honored with `consult-line` together with `consult-preview-at-point-mode` (in `completion-list-mode`)_** 

![2024-04-09_08-27](https://github.com/minad/consult/assets/42199775/2809ff45-5f14-46c7-8909-29159e312770)

**Comment by emacksnotes** [comment]
See related bug/discussion [Need a recipe for annotating consult-line ( consult-grep etc) candidte buffer with which-function]( https://github.com/minad/consult/discussions/993#discussioncomment-9055463)

**Reply by minad** [reply]
Let's close this in favor of #993. No reason for two discussions which are closely related.
